TESTING.md

```diff
@@ -66,15 +66,37 @@ For manual testing, you can use the utility script [scripts/run-external.sh](scr
 ./scripts/run-external.sh -c
 ```
 
-Before running automated end-to-end tests, you need run the following command to make images and load it in your local cluster (for KinD cluster):
+### Building Images and Loading them into your cluster
+
+#### Using docker with Kind
+
+Before running automated end-to-end tests, you need run the following command to make images and load it in your local cluster:
 
 ```
 make image
 
 for n in "prometheus-operator" "prometheus-config-reloader" "admission-webhook"; do kind load docker-image "quay.io/prometheus-operator/$n:$(git rev-parse --short HEAD)"; done;
 ```
 
-Then to run the automated end-to-end tests, run the following command:
+#### Using podman with Kind
+
+When running kind on MacOS using podman, it is recommended to create podman machine 4 CPUs and 8 GiB memory. Less resources might cause end to end tests fail because of lack of resources for cluster.
```

Comment:
```suggestion
When running kind on MacOS using podman, it is recommended to create podman machine 4 CPUs and 8 GiB memory. Less resources might cause end to end tests fail because of lack of resources in cluster.
```